src/python/pants/backend/project_info/rules/fast_dependencies.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+from typing import Set
+
+from twitter.common.collections import OrderedSet
+
+from pants.base.specs import AscendantAddresses, DescendantAddresses, SingleAddress, Specs
+from pants.build_graph.address import Address
+from pants.engine.console import Console
+from pants.engine.goal import Goal
+from pants.engine.legacy.graph import HydratedTargets, TransitiveHydratedTargets
+from pants.engine.rules import console_rule
+from pants.engine.selectors import Get
+
+
+class Dependencies(Goal):
```

Comment:
This should also subclass `LineOriented`, as `List` does:

https://github.com/pantsbuild/pants/blob/5fcb49245701c85243e49e981d6478df3571e9a2/src/python/pants/rules/core/list_targets.py#L13

This registers two options for us: `--output-file` and `--sep`. It also gives us this way of writing to the console so that we respect these options properly:

https://github.com/pantsbuild/pants/blob/5fcb49245701c85243e49e981d6478df3571e9a2/src/python/pants/rules/core/filedeps.py#L61-L64